File: Ejercicio_3/ejercicio3.c
// # Escriba un algoritmo en pseudo código, para verificar si un número es primo

#include <stdio.h>

// Funcion para determinar si un numero es primo

void prime_num(int n)
{
    if (n == 1)
    {
        printf("\nEl numero no es primo!\n");
        return;
    }
    else
    {
        for (int i = 2; i < n; i++)
        {
            if (n % i == 0)
            {
                printf("\nEl numero no es primo!\n");
                return;
            }
        }
        printf("\nEl numero es primo!\n");
    }
}

void main()
{
    int num;
    printf("\nIngrese el numero: ");
    scanf("%d", &num);
    prime_num(num);
}

File: Ejercicio_3/ejercicio3.py
# Escriba un algoritmo en pseudo código, para verificar si un número es primo

# Funcion para determinar si un numero es primo

def prime_num(n):
    if (n == 1):
        print("\nEl numero no es primo!\n")
        return None
    else:
        for i in range(2, n):
            if (n % i == 0):
                print("El numero no es primo!\n")
                return None

    print("\nEl numero es primo!\n")

num = int(input("\nIngrese el numero: "))
prime_num(num)